gensim/api/__init__.py

```diff
@@ -0,0 +1,174 @@
+from __future__ import print_function
```

Comment:
rename this file as `downloader.py` and move to library root `gensim/downloader.py`, also merge file with cli and this